### PR TITLE
add new family to test_party_add() to ensure none are skipped

### PR DESCRIPTION
**Pull request by cjhopp**
Adding bogus Family from scratch to original test_party_add function. Not sure if this is the cleanest way to do it....but it was the first that came to mind.

### CODE REVIEW

File: eqcorrscan/tests/match_filter_test.py [hunk 1]
```diff
@@ -682,10 +682,20 @@ def test_party_add(self):
         test_party += test_family
         self.assertEqual(len(test_party), 5)
         test_slice = test_party[0:2]
+        # Add new fam with fake det to ensure unique families aren't missed
+        new_family = Family(template=self.tribe[-1].copy(),
```
Comment (calum-chamberlain) [inline eqcorrscan/tests/match_filter_test.py]: Putting the new-line after Family( might make this cleaner, e.g.
```python
new_family = Family(
    template=self.tribe[-1].copy(), ... )
```

**Comment by calum-chamberlain** [comment]
Closes #102

**Comment by cjhopp** [comment]
Consistent Mac and Windows timeouts. Linux passes. Merge anyways?

**Comment by calum-chamberlain** [comment]
+1 go for it. I'm making tests faster in the Xarray branch, figured out how to get them working well in parallel

Calum Chamberlain
Postdoctoral Fellow, Seismology
Victoria University of Wellington
Sent from mobile.

________________________________

**Pull request merged**